src/Simulation/Simulators.Type1.Tests/Tests.Microsoft.Quantum.Simulators.Type1.csproj

```diff
@@ -0,0 +1,57 @@
+﻿<Project Sdk="Microsoft.Quantum.Sdk/0.12.20082705-beta">
```

Comment:
Same question as the props file - this is almost identical to the Type2 csproj, is it possible to factor more of it out into a common file?

Reply:
Honestly, while I think it could, I think at some point the reduced duplication is a diminishing return for the increased complexity. Eventually, I hope we can get to the point where all the simulators are unified into a single project, but for now I think I prefer to have them explicitly separate.

Reply:
I'll see if the approach you suggested for the props file will also work here. Thanks!